### PR TITLE
refactor: enhance i18n type annotations for better developer experience

### PR DESCRIPTION
**Pull request by zjy365**

### CODE REVIEW

File: frontend/providers/applaunchpad/src/pages/app/edit/components/QuotaBox.tsx [hunk 1]
```diff
@@ -27,9 +27,10 @@ const sourceMap = {
 
 const QuotaBox = () => {
   const theme = useTheme();
-  const { t } = useTranslation();
+  const { t } = useTranslation('common');
   const { userQuota, loadUserQuota } = useUserStore();
   useQuery(['getUserQuota'], loadUserQuota);
+  console.log(userQuota);
```
Comment (zzjin) [inline frontend/providers/applaunchpad/src/pages/app/edit/components/QuotaBox.tsx:33]: debug console.log need removed.

**Comment by zjy365** [comment]
image

**Comment by zjy365** [comment]
image

**Comment by zjy365** [comment]
image

File: frontend/desktop/src/hooks/useDriver.tsx [hunk 2]
```diff
@@ -304,7 +304,7 @@ export default function useDriver({ openDesktopApp }: { openDesktopApp: any }) {
         transform="translate(-50%, -50%)"
       >
         <Box fontSize={'32px'} fontWeight={500}>
-          {t('Hello, welcome')}
+          {t('common:hello,_welcome')}
```
Comment (zzjin) [inline frontend/desktop/src/hooks/useDriver.tsx:307]: comma need to to removed.

**Review by zzjin — APPROVED** [review]

**Pull request merged**